src/main/java/com/algorand/algosdk/v2/client/indexer/LookupAccountTransactions.java

```diff
@@ -0,0 +1,160 @@
+package com.algorand.algosdk.v2.client.indexer;
+
+import java.util.Date;
+
+import com.algorand.algosdk.crypto.Address;
+import com.algorand.algosdk.util.Encoder;
+import com.algorand.algosdk.v2.client.common.Client;
+import com.algorand.algosdk.v2.client.common.Query;
+import com.algorand.algosdk.v2.client.common.QueryData;
+import com.algorand.algosdk.v2.client.common.Response;
+import com.algorand.algosdk.v2.client.model.Enums;
+import com.algorand.algosdk.v2.client.model.TransactionsResponse;
+
+
+/**
+ * Lookup account transactions. 
+ * /accounts/{account-id}/transactions 
+ */
+public class LookupAccountTransactions extends Query {
+
+	private Address accountId;
+
+	/**
+	 * @param accountId account string 
+	 */
+	public LookupAccountTransactions(Client client, Address accountId) {
+		super(client, "get");
+		this.accountId = accountId;
+	}
+
+	/**
+	 * Include results after the given time. Must be an RFC 3339 formatted string. 
+	 */
+	public LookupAccountTransactions afterTime(Date afterTime) {
+		addQuery("after-time", afterTime.toInstant().atZone(java.time.ZoneOffset.ofHours(0)).toString());
```

Comment:
This isn't available in Java 7 :(

Reply:
We already import threeten, you should go ahead and switch to that: https://www.threeten.org/threetenbp/

Reply:
:-( Fixed with standard java libs.